compliant_social_django/storage.py

```diff
@@ -188,9 +188,15 @@ def refresh_token(self, strategy, *args, **kwargs):
             # break the access token and refresh token out of the extra data
             self.actual_access_token = extra_data.pop('access_token', None)
             self.actual_refresh_token = extra_data.pop('refresh_token', self.actual_refresh_token)
-            if self.set_extra_data(extra_data):
+            was_updated = self.set_extra_data(extra_data)
+            if was_updated or self.revoked:
```

Comment:
Wait, so if it wasn't updated but it had previously been revoked, we're now reactivating it? Is that right? Should that maybe be an `and`?

Reply:
As discussed - we have removed the condition to improve clarity of this code, even if by just a little bit.